packages/next/src/build/utils.ts

```diff
@@ -824,9 +823,10 @@ export function printCustomRoutes({
       })
       .join('\n')
 
-    print(routesStr, '\n')
+    print(`${routesStr}\n`)
```

Comment:
Note that `print` is just a `console.log`.
It was adding a single space after `routeStr`, think of `console.log('a', 'b')`.